Omise.Tests/Resources/ChargeResourceTest.cs

```diff
@@ -83,6 +84,7 @@ public void TestCreateChargeRequest()
                 @"""amount"":244884," +
                 @"""currency"":""thb""," +
                 @"""description"":""Test Charge""," +
+                @"""expires_at"":""2023-08-08T17:00:00Z""," +

```

Comment:
Can we make this test to do not fail in the future?

Reply:
it will not fail, as the test is fixed on this date